src/Zendesk/Types.hs

```diff
@@ -177,6 +224,29 @@ data TicketInfo = TicketInfo
     , ticketStatus :: !Text        -- ^ The status of the ticket
     } deriving (Eq, Show)
 
+
+-- TODO(ks): Newtype!
+instance Arbitrary TicketURL where
+    arbitrary = do
+        protocol    <- elements ["http://"]
```

Comment:
`protocol` only has 1 element so we could just write it as `let protocol = "http://"` or we can add `https://` as one of the choices! so it would be `elements ["http://", "https://"]`

Reply:
Yes, that is an option I left. We will fix this once we start wrapping it in a newtype.